packages/eslint-config/package.json

```diff
@@ -25,16 +25,20 @@
     "readme": "ts-node ../../scripts/replace-common-readme.ts"
   },
   "dependencies": {
+    "@eslint/js": "^9.11.1",
+    "@stylistic/eslint-plugin": "^2.8.0",
     "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-import": "^2.30.0",
     "eslint-plugin-jsdoc": "^50.2.5",
+    "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-regex": "^1.10.0",
     "eslint-plugin-unused-imports": "^4.1.4",
-    "typescript-eslint": "^7.18.0",
-    "@eslint/js": "^9.11.1"
+    "typescript-eslint": "^7.18.0"
   },
   "peerDependencies": {
     "eslint": "^8.0.0"
+  },
+  "devDependencies": {
+    "@stylistic/eslint-plugin-migrate": "^2.8.0"
```

Comment:
[q] Is this needed? Briefly looking [here](https://eslint.style/guide/migration#eslint-migrate-plugin) it says we need to add comments at the top of the config file to automatically migrate rules.

Reply:
no not needed anymore, also just realized this and removed it 😊